action.yml

```diff
@@ -1,18 +1,22 @@
-name: 'AWS Secrets Manager GitHub Action'
-author: 'AWS Secrets Manager'
-description: 'GitHub action for retrieving secrets from AWS Secrets Manager'
+name: "AWS Secrets Manager GitHub Action"
+author: "AWS Secrets Manager"
+description: "GitHub action for retrieving secrets from AWS Secrets Manager"
 branding:
-  icon: 'cloud'
-  color: 'orange'
+  icon: "cloud"
+  color: "orange"
 inputs:
   secret-ids:
-    description: 'One or more secret names, secret ARNs, or secret prefixes to retrieve'
+    description: "One or more secret names, secret ARNs, or secret prefixes to retrieve"
     required: true
   parse-json-secrets:
-    description: '(Optional) If true, JSON secrets will be deserialized, creating a secret environment variable for each key-value pair.'
+    description: "(Optional) If true, JSON secrets will be deserialized, creating a secret environment variable for each key-value pair."
```

Comment:
Idem here, is there a mismatch between existing code and ESLint rules? Otherwise revert these formatting changes.

Reply:
Fixed